Vocabulary/Suffixes.md

```diff
@@ -36,6 +36,7 @@
 | -ku | establishes verb as uncertain/questioning | — |
 | -do | establishes verb as imperative | "should", "ought to" |
 | -kuka | establishes verb as politely questioning/requesting | "please" |
+| -zim | establishes verb as infinitive | "to ___" |
```

Comment:
@realblobii Shouldn't this be a temporal suffix? It actually modifies the tense of the verb by changing it to infinitive form...

Reply:
yes my bad ill fix that quickly